CRM/Contribute/Form/Contribution.php

```diff
@@ -1589,7 +1585,7 @@ protected function submit($submittedValues, $action, $pledgePaymentID) {
     }
 
     if (!isset($submittedValues['total_amount'])) {
-      $submittedValues['total_amount'] = CRM_Utils_Array::value('total_amount', $this->_values);
+      $submittedValues['total_amount'] = CRM_Utils_Array::value('total_amount', $this->_values) - CRM_Utils_Array::value('tax_amount', $this->_values);
```

Comment:
I'm not sure but we're in the form/Contribution.php so not worried about too many decimals here. I'll make sure I test with some numbers that did previously introduce rounding issues before I fixed that issue earlier this year.